### PR TITLE
chore: bump SDK v0.45.12

### PR DESCRIPTION
**Pull request by mmulji-ic**
Closes issue #2083

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -29,6 +28,7 @@ require (
 require (
 	cosmossdk.io/math v1.0.0-beta.4
 	github.com/ory/dockertest/v3 v3.9.1
+	github.com/tendermint/tendermint v0.34.24
```
Comment (glnro) [inline go.mod]:
```suggestion
```

File: go.mod [hunk 2]
```diff
@@ -299,6 +295,8 @@ replace (
 
 	// use cosmos style protobufs
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+	// Use Informal Systems fork of Tendermint
 	github.com/tendermint/tendermint => github.com/informalsystems/tendermint v0.34.24
```
Comment (mpoke) [inline go.mod:303]: Do we still need this?

File: go.mod [hunk 3]
```diff
@@ -20,7 +20,6 @@ require (
 	github.com/spf13/viper v1.14.0
 	github.com/strangelove-ventures/packet-forward-middleware/v3 v3.1.1
 	github.com/stretchr/testify v1.8.1
-	github.com/tendermint/tendermint v0.34.24
```
Comment (glnro) [inline go.mod]: No need to remove here

**Review by glnro — CHANGES_REQUESTED** [review]

File: go.mod (hunk 3 above)
Comment (mpoke) [reply go.mod]: Why? I though that we don't need to directly import TM.

File: go.mod (hunk 2 above)
Comment (glnro) [reply go.mod:303]: We need to keep the replace at minimum...gaia doesn't need tendermint directly.

File: go.mod (hunk 3 above)
Comment (glnro) [reply go.mod]: We actually do, other dependencies require it.

File: go.mod [hunk 4]
```diff
@@ -285,11 +284,8 @@ replace (
 	// Use cosmos keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 
-	// dragonberry
-	github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
```
Comment (glnro) [inline go.mod:289]: Per the [SDK's release notes](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.12), it's advised to also bump `github.com/confio/ics23/go v0.9.0`...while I see this has been indirectly required, given the nature of the version bump and the SDK's recommendation we should explicitly require this.

**Review by glnro — CHANGES_REQUESTED** [review]

File: go.mod (hunk 3 above)
Comment (mmulji-ic) [reply go.mod]: This was the go mod action, but have updated this time. 
We shouldn't mangle too much what go mod kicks out.

File: go.mod (hunk 4 above)
Comment (mmulji-ic) [reply go.mod:289]: SDK says:
# Can be deleted from go.mod
replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
Instead, github.com/confio/ics23/go must be bumped to v0.9.0.

Which is what I've done.

File: go.mod (hunk 1 above)
Comment (mmulji-ic) [reply go.mod]: This was the go mod action, but have updated this time. 
We shouldn't mangle too much what go mod kicks out.

File: go.mod (hunk 4 above)
Comment (glnro) [reply go.mod:289]: It's being required indirectly [here](https://github.com/cosmos/gaia/pull/2088/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R72)...we should explicitly require this version.

**Review by glnro — APPROVED** [review]

**Review by mpoke — APPROVED** [review]

**Review by sainoe — APPROVED** [review]

**Pull request merged**